sdk/analysisservices/Azure.ResourceManager.Analysis/tests/TestCase/AnalysisServiceTests.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.Analysis.Models;
+using Azure.ResourceManager.Models;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.Analysis.Tests
+{
+    public class AnalysisServiceTests : AnalysisManagementTestBase
```

Comment:
Please generate the samples first, then refer the generated samples to add more tests for all resources.

Reply:
fixed